packages/flame/lib/src/game/game.dart

```diff
@@ -34,6 +35,10 @@ abstract mixin class Game {
   /// functionality in Flutter.
   late final GestureDetectorBuilder gestureDetectors =
       GestureDetectorBuilder(refreshWidget)..initializeGestures(this);
+  
+  /// Set by the PointerMoveDispatcher to receive mouse events from the
+  /// game widget.
+  void Function(PointerHoverEvent event)? mouseDetector;
```

Comment:
sadly this does not seem to be possible to be included under the gestureDetectors - events don't fire as intended.